apps/paymaster-proxy/package.json

```diff
@@ -11,9 +11,9 @@
     "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
     "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn",
     "loadtest": "pnpm dlx artillery run loadtest/basic-loadtest.yml",
-    "docker:services:up": "nx run api-key-service:docker:up && docker compose up -d redis postgres",
-    "docker:up": "nx run api-key-service:docker:up && docker compose up -d redis postgres paymaster-proxy-app",
```

Comment:
running in separate docker compose stack makes networking a bit harder to setup. more maintainable to run all containers in the same stack